<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="8"
           failures="3"
           errors="0"
           skipped="1"
           time="11.780">
  <properties />
            <testcase name="inquiryInputInfromationValue" time="0.624" classname="com.inquiry.tests.Report_VerifyTest">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	Address Values are not Matching expected [FLAT NO.404 BLOCK C MAHAVEER REGAL HOODI BANGALORE  560066 KA] but found [FLAT NO.404 BLOCK C MAHAVEER REGAL HOODI BANGALORELORE 560066 KA]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	Address Values are not Matching expected [FLAT NO.404 BLOCK C MAHAVEER REGAL HOODI BANGALORE  560066 KA] but found [FLAT NO.404 BLOCK C MAHAVEER REGAL HOODI BANGALORELORE 560066 KA]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.inquiry.tests.Report_VerifyTest.inquiryInputInfromationValue(Report_VerifyTest.java:165)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="headerInformation_Field_ValueTest" time="0.577" classname="com.inquiry.tests.Report_VerifyTest">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	issue date are not matched expected [13-11-2018] but found [15-10-2018],
	request date not matched expected [13-11-2018] but found [15-10-2018],
	report id not matched expected [TEST181113CR256614075] but found [TEST181015CR241084568]"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	issue date are not matched expected [13-11-2018] but found [15-10-2018],
	request date not matched expected [13-11-2018] but found [15-10-2018],
	report id not matched expected [TEST181113CR256614075] but found [TEST181015CR241084568]
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.inquiry.tests.Report_VerifyTest.headerInformation_Field_ValueTest(Report_VerifyTest.java:81)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
            <testcase name="test_MemmberAccountInformation" time="2.369" classname="com.inquiry.tests.Report_VerifyTest">
                <failure type="java.lang.AssertionError"
                  message="The following asserts failed:
	Realtionship_Field field is not matching. expected [Relationship #1] but found []"
              >
<![CDATA[java.lang.AssertionError: The following asserts failed:
	Realtionship_Field field is not matching. expected [Relationship #1] but found []
  org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
  com.inquiry.tests.Report_VerifyTest.test_MemmberAccountInformation(Report_VerifyTest.java:305)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
  
            <testcase name="inquiryInputInfromationfield" time="0.0" classname="com.inquiry.tests.Report_VerifyTest">
          <skipped />
    </testcase>
  
            <testcase name="test_DisclaimerPresence" time="0.341" classname="com.inquiry.tests.Report_VerifyTest" />
                <testcase name="test_MemmberAccountInformationTradeLineField" time="5.220" classname="com.inquiry.tests.Report_VerifyTest" />
                <testcase name="headerInformation_FieldTest" time="0.548" classname="com.inquiry.tests.Report_VerifyTest" />
                <testcase name="testSummary_Section" time="2.101" classname="com.inquiry.tests.Report_VerifyTest" />
      
</testsuite>
